### PR TITLE
various typography

### PR DESCRIPTION
**Pull request by friism**
this is purely docs/README-related

### CODE REVIEW

File: docs/kernels.md [hunk 1]
```diff
@@ -37,7 +37,6 @@ In summary, LinuxKit offers a choice of the following kernels:
 - [linuxkit/kernel-centos](https://hub.docker.com/r/linuxkit/kernel-centos/): Selected CentOS kernels.
 - [linuxkit/kernel-fedora](https://hub.docker.com/r/linuxkit/kernel-fedora/): Selected Fedora kernels.
 
-
```
Comment (rn) [inline docs/kernels.md]: why are you deleting these empty lines? The file mostly uses two new-line before a section heading to make it visually clearer that a new section starts (though there are a couple of case where this is not followed).

Comment (friism) [reply docs/kernels.md]: Those extra newlines waste valuable bandwidth when doing git-pull, etc.. I can make it consistent at 2 newline if you prefer though.

Comment (friism) [reply docs/kernels.md]: @rn fixed

**Comment by justincormack** [comment]
Commits need to be signed off, we can't review this otherwise.

**Pull request closed without merging**